timescaledb/tutorials/building-a-custom-timescaledb-dashboard.md

```diff
@@ -0,0 +1,326 @@
+
+Notice in this sample chunk from our test dataset that `before_compression_total_bytes` is 10 times bigger than `after_compression_total_bytes`. We save more than 90% of disk space with compression!
+
+```sql
+SELECT * FROM  chunks_with_compression;
+...
+-[ RECORD 96 ]-----------------+-----------------------
+chunk_name                     | _hyper_2_37_chunk
+chunk_schema                   | _timescaledb_internal
+hypertable_schema              | public
+hypertable_name                | conditions
+range_start                    | 2021-05-27 00:00:00+00
+range_end                      | 2021-06-03 00:00:00+00
+before_compression_total_bytes | 90112
+after_compression_total_bytes  | 8192
+```
+
+
+## Setting up your database
+
+In this project, we use data generated by our tutorial on [Simulating IoT sensor data][iot-tutorial]. This data will result in a simple schema and data that mimics a number of IoT sensors with information on time, device, and temperature.
+
+If you follow the tutorial, you will get a table named conditions, which stores the temperature of fake devices over time.
+
+Here’s a set of commands to create the table and generate some sample data:
+
+```sql
+CREATE TABLE conditions (
+      time TIMESTAMPTZ NOT NULL,
+      device INTEGER NOT NULL,
+      temperature FLOAT NOT NULL,
+      PRIMARY KEY(time, device)
+);
+
+SELECT * FROM create_hypertable('conditions', 'time', 'device', 3);
+
+INSERT INTO conditions
+  SELECT time, (random()*30)::int, random()*80 - 40
+  FROM generate_series(TIMESTAMP '2020-01-01 00:00:00',
+                       TIMESTAMP '2020-01-01 00:00:00' + INTERVAL '1 month',
+             INTERVAL '1 min') AS time;
+
+```
+
+
+## Connecting to our database and retrieving metrics
+
+One of the most trivial aspects of a backend application is to protect the database and expose only the information that is accessible to the authorized user. The Hasura GraphQL Engine solves this issue providing GraphQL APIs over new or existing PostgreSQL databases. Allowing you to create permission rules and dynamically expand your database resources.
+
+Now that we have a database example, we can jump into the [Hasura cloud][hasura-cloud] to connect the resources that we want to expose through the GraphQL.
+
+We chose Hasura because of the flexibility to simply connect our TimescaleDB database and quickly expose the tables, views, and functions we needed in a few clicks. We’re not going to dive into all the steps to set up a new data source on Hasura as the wizard will guide you there.
+
+We’re going to use two types of operations:
+
+* Queries and subscriptions: watch a specific query and keep pulling data updates to the client. We’re going to "subscribe" to the chunks metadata’.
+* Mutation: convention for operations that are writing data. We’ll map the compression and decompression actions as mutations.
```

Comment:
```suggestion
We’re going to use two types of operations:

* Queries and subscriptions: watch a specific query and keep pulling data updates to the client. In this example, you subscribe to the chunks' metadata.
* Mutation: convention for operations that write data. In this example, you map the compression and decompression actions as mutations.
```

"convention"?

Reply:
You rock @Loquacity! I'm learning a lot from your reviews and suggestions ❤️